logstash-core/lib/logstash/compiler.rb

```diff
@@ -19,31 +19,14 @@
 
 java_import org.logstash.config.ir.PipelineIR
 java_import org.logstash.config.ir.graph.Graph
+java_import org.logstash.config.ir.ConfigCompiler
```

Comment:
nitpick: a little off-topic here, but could we move the imports into `class Compiler`' scope